enumerables.rb

```diff
@@ -0,0 +1,107 @@
+# Enumerable methods
+
+module Enumerable
+  # my_each
+
+  def my_each
+    length.times { |i| yield(self[i]) }
+    self
+  end
+
+  # end of my_each
+
+  # my_each_with_index
+  def my_each_with_index
+    length.times { |i| yield(self[i], i) }
+    self
+  end
+
+  # end of my_each_with_index
+
+  # my_select
+
+  def my_select
+    arr = []
+    my_each { |i| arr.push(i) if yield(i) }
+    arr
+  end
```

Comment:
* [ ] should have working `my_select` identical and returns the same thing as ruby's [`select`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-select). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_select.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))

#### my_select

- [ ] my_select returns an enumerator if no block is given

- Yours is giving a LocalJumpError
![image](https://user-images.githubusercontent.com/52209067/81132358-5fa0b000-8f0b-11ea-8238-bdd85b1647fb.png)

- Real select
![image](https://user-images.githubusercontent.com/52209067/77681510-0bdd9700-6f5b-11ea-9340-ac116b7ed62e.png)